packages/node_modules/@ciscospark/plugin-memberships/src/memberships.js

```diff
@@ -2,7 +2,21 @@
  * Copyright (c) 2015-2017 Cisco Systems, Inc. See LICENSE file.
  */
 
-import {SparkPlugin, Page} from '@ciscospark/spark-core';
+import { SparkPlugin, Page } from '@ciscospark/spark-core';
+import {
+  WEBEX_API_EVENTS,
+  API_ACTIVITY_TYPE,
+  API_ACTIVITY_VERB,
+  constructHydraId,
+  hydraTypes
+} from '@ciscospark/common';
+
+const debug = require('debug')('memberships');
+
+const MEMBERSHIP_CREATED = 'created';
+const MEMBERSHIP_DELETED = 'deleted';
+const MEMBERSHIP_UPDATED = 'updated';
+const CONVERSATION_ACTIVITY = 'event:conversation.activity';
```

Comment:
We may want to put this in a common file now that we're using it in multiple places.

Reply:
Agree.